_posts/2023-06-28-Remilia-interview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Interview wth contributor Remilia Scarlet"
+author: beta-zilliani,Remilia
+summary: "<b>Music playback projects in Crystal, games engines, and a bit of Lisp"
+---
+<https://remilia.sdf.org/>
+
+<https://www.youtube.com/watch?v=UX2in-whUik>
+
+_Conversation between Beta Ziliani and Remilia Scarlet_
+## A little about you. Who's Remilia?
```

Comment:
Interview questions as headlines feel weird. Especially for longer ones that span multiple lines. A better approach would be to use normal paragraphs set in bold. That's the typical format for print interviews.

Reply:
The "fix it later" approach doesn't work as well as in source repositories. This content is going to be published immediately after merging and we should make sure to polish it right away.
What we do about the old post is a different story.

Reply:
In the current setting adding one level of nesting (`###`) does the trick pretty well:
<img width="945" alt="image" src="https://github.com/crystal-lang/crystal-website/assets/5183634/40506b8b-acad-416f-8baf-46e5282b6523">

And provides the vertical space to separate Q from A. WDYT?

Reply:
I don't see how this would be any better than the bold paragraph I suggested.